### DISCUSSION TITLE
`javascript/object` format question

### DISCUSSION THREAD

**Discussion by personable** [amzn/style-dictionary#1178] [Q&A] [answered]
Hi, all! Love this library so far. It's fantastic. This is probably a dumb question, but when I use the predefined `javascript/object` format on my tokens, I'm expecting the output to look like it does in the docs:

![image](https://github.com/amzn/style-dictionary/assets/1197038/b665beea-69d7-4463-a31c-a566863e2be5)

However, I get this instead.

<img width="720" alt="image" src="https://github.com/amzn/style-dictionary/assets/1197038/4b6c62d1-6bc6-49f6-917c-f808d81e61f6">

Does anyone know how I can get the quotes around the keys to go away?

**Comment by jorenbroekema** [comment, marked as answer]
[This format is using `JSON.stringify`](https://github.com/amzn/style-dictionary/blob/v4/lib/common/formats.js#L393) to stringify the tokens object into what you see in your output.
It's not always easy to determine whether or not "" is required for an object property key, my suggestion would be to [create a custom format](https://v4.styledictionary.com/reference/hooks/formats/#custom-formats) that does the same thing but that also runs [Prettier](https://prettier.io/docs/en/api) on the output which should get rid of any redundant quotes around property keys.

**Reply by personable** [reply]
Thanks @jorenbroekema  - that's where I was heading, but wanted to make sure I wasn't missing something dumb (more of a designer who codes than a dev). Thanks again
